addons/bus/static/tests/websocket_service_tests.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module */
+
+import { websocketService } from "@bus/websocket_service";
+
+import { registry } from "@web/core/registry";
+import { patchWebsocketWorkerWithCleanup } from "@web/../tests/helpers/mock_websocket";
+import { nextTick } from "@web/../tests/helpers/utils";
+import { createWebClient } from "@web/../tests/webclient/helpers";
+
+QUnit.module("Websocket Service", () => {
+    QUnit.test("displays reconnect notification according to worker events", async (assert) => {
+        assert.expect(3);
+
+        const { worker } = patchWebsocketWorkerWithCleanup();
+        registry.category("services").add('websocketService', websocketService);
+
+        await createWebClient({});
+        worker.broadcast('reconnecting', { closeCode: 1006 });
```

Comment:
I have no clue what's `closeCode: 1006`, a comment would be appreciated, especially if this is relevant information here (which looks it is, I see `closeCode: 4001` below that I don't understand neither)